bin/dynamodb_check

```diff
@@ -1,15 +1,26 @@
 #!/usr/bin/env python2
 import argparse
+from urlparse import urlparse
 import boto3
 from botocore.client import Config
 
 
 def main(args):
+    parts = urlparse(args.url)
+
     config = Config(connect_timeout=args.timeout,
                     read_timeout=args.timeout)
 
+    if parts.port:
+        port = parts.port
+    else:
+        port = 8000
+
     client = boto3.client("dynamodb",
-                          endpoint_url=args.url,
+                          region_name="eu-west-1",
+                          aws_access_key_id="fake",
```

Comment:
My thinking is this is _only_ for local DynamoDB. You can check real dynamo DB with:

```
await cmd -- aws dynamodb list-tables
```

The difference is we don't have to support all the various things that AWS cli can do _and_ now we can await any various AWS api calls.